iaso/api/setup_account.py

```diff
@@ -52,6 +51,21 @@ def create(self, validated_data):
             user_manual_path=validated_data.get("user_manual_path"),
         )
 
+        # Create a setup_account project with an app_id represented by the account name
+        initial_project = Project.objects.create(
+            name=validated_data["account_name"] + " project", account=account, app_id=validated_data["account_name"]
```

Comment:
I would probably make sure that the account name doesn't have spaces in its name for the app_id:
```suggestion
            name=validated_data["account_name"] + " project", account=account, app_id=validated_data["account_name"].replace(' ', '.')
```